Yurii sv

YuriiSvishchev opened the pull request:
Я создаю ПР для проверки практической работы.
Изначально ПР создать не получилось, т.к. сразу работал в ветки main.
Поэтому пришлось всё удалить, заново форкнуть проект, создать в нём новую ветку и в эту ветку скопировать ранее написанный  код.

YuriiSvishchev committed:
Начало работы

YuriiSvishchev committed:
Завершение работы

Charmandik commented on src/main/java/Main.java:
```diff
@@ -1,6 +1,90 @@
+                }
+            }
+            double fullPrice = 0.0;
+            StringBuilder str = new StringBuilder();
```
Старайся сразу писать полные имена переменных. Это важно, потому что приложения, которые в конечном итоге ты будешь писать - будут состоять из 100 или из 1000 или даже из 10000 различных классов и просто невозможно запомнить все эти сокращения, которые кто-то выдумал, пока писал какой-либо из классов и читать такой код в разы сложнее

Charmandik commented on src/main/java/Main.java:
```diff
@@ -1,6 +1,90 @@
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+
+            Scanner scanner = new Scanner(System.in);
```
после использования сканера хорошей практикой будет закрыть его с помощью функции scanner.close(), это позволит избежать возможных утечек памяти

YuriiSvishchev committed:
Исключена возможность добавления отрицательной стоимости товара.
Изменено имя переменой на StringBuilder "check", вместо  StringBuilder "str".
Добавлена функция scanner.close().
